index.html

```diff
@@ -254,22 +283,25 @@ <h4>professional art printing data</h4>
 
         <section class="skills">
           <div class="all-skills">
-            <div class="skills-details">
-              <h2>About Me</h2>
-              <p>
-                Hello I'm a software developer! I can help you build a product ,
-                feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don't hesitate to contact me.
-              </p>
-              <div class="details-btn">
-                <a
-                  href="./assets/images/CV - Blessing Oghenetega Ekiugbo (1).pdf"
-                  download
-                  >Get my Resume</a
-                >
+            <div class="about-skills-details">
+              <div class="skills-details">
+                <h2>About Me</h2>
+                <p>
+                  Hello I'm a software developer! I can help you build a product
+                  , feature or website Look through some of my work and
+                  experience! If you like what you see and have a project you
+                  need coded, don't hesitate to contact me.
+                </p>
+                <div class="details-btn">
+                  <a
+                    href="./assets/images/CV - Blessing Oghenetega Ekiugbo (1).pdf"
+                    download
+                    >Get my Resume</a
+                  >
+                </div>
               </div>
             </div>
+
             <div class="tech-stack">
               <ul>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct value to `align-items` because all `li` items postion `center`❌, it should be `left` ✔️ 

![image](https://user-images.githubusercontent.com/63915024/220963184-b5598b3c-4971-4786-b8e5-764eb360184d.png)